packages/apptive_grid_core/lib/src/model/form/form_block.dart

```diff
@@ -0,0 +1,78 @@
+/// A freeform block inside of a form
+class FormBlock {
```

Comment:
Think this should have "Text" somewhere in there

Reply:
I renamed it.